### PR TITLE
Quality of life features for mobile and changes to MusicButton.js file

### PR DESCRIPTION
**Pull request by sakulstar**
**Changes:**

1. Renamed MusicButton.js to YoutubeButton.
2. Added link variable to MusicButton.
3. Changed copyright paragraph text to change size for mobile (414px width and below).
4. Changed top right discord button to hide for small mobile (414px width and below).

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -2,7 +2,11 @@
 module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      screens: {
+        'mobile': {'max': '414px'},
```
Comment (dylhack) [inline tailwind.config.js:7]: Tailwind provides responsive breakpoints https://tailwindcss.com/docs/responsive-design#overview

**Review by dylhack — COMMENTED** [review]
Just one nitpick, leaving further review to @CrimpsonSlopers

File: tailwind.config.js (hunk 1 above)
Comment (sakulstar) [reply tailwind.config.js:7]: Not quite sure what you changed?

**Review by CrimpsonSlopers — APPROVED** [review]
Yep Code

**Pull request merged**